devtools-ci-index.yaml

```diff
@@ -2455,6 +2455,10 @@
             - text:
                 credential-id: "{osio_token}"
                 variable: REFRESH_TOKEN
+            - username-password-separated:
+                credential-id: "{osio_creds}"
```

Comment:
You are introducing a new variable `{osio_creds}` in the template, without defining a default, and there are some other jobs in this file that use this template that haven't defined this variable:

https://github.com/openshiftio/openshiftio-cico-jobs/blob/1bb2fc2b4ee26480015be302abb13a8cc0ee3937/devtools-ci-index.yaml#L4092-L4113

Reply:
@Raunak1203 can we add these creds to vault? we are moving all creds from CICO to vault
I do not have access to CICO credentials so I don't know if `b310559d-91c5-469c-bb35-fc9b8333e619` and `e53d9045-2500-498a-936f-3ecf323af800` exists at all

Reply:
@riuvshin Housekeeping issue for CICO credentials https://gitlab.cee.redhat.com/dtsd/housekeeping/issues/2129# 
and How can we add these creds to vault ?

Reply:
ok seems it is in place. so let's reuse it, and I will move them all to vault later.

Reply:
Thanks @riuvshin